src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -6,17 +6,12 @@
 import static frc.team2412.robot.Subsystems.SubsystemConstants.INTAKE_ENABLED;
 import static frc.team2412.robot.Subsystems.SubsystemConstants.SHOOTER_ENABLED;
 
-import frc.team2412.robot.subsystem.ClimbSubsystem;
-import frc.team2412.robot.subsystem.DrivebaseSubsystem;
-import frc.team2412.robot.subsystem.IndexSubsystem;
-import frc.team2412.robot.subsystem.IntakeSubsystem;
-import frc.team2412.robot.subsystem.ShooterSubsystem;
-import frc.team2412.robot.subsystem.ShooterVisionSubsystem;
+import frc.team2412.robot.subsystem.*;
 import io.github.oblarg.oblog.Loggable;
 
 public class Subsystems implements Loggable {
     public static class SubsystemConstants {
-        public static final boolean CLIMB_ENABLED = true;
+        public static final boolean CLIMB_ENABLED = false;
```

Comment:
Why is climb disabled?

Reply:
because we dont want to bonk button by mistake when testing

Reply:
Please remove this change.  We don't want to get to competition with a disabled climb.